### PR TITLE
Add the missing lua files to OpenRA.sln

### PR DESCRIPTION
**Pull request by abcdefg30**

### CODE REVIEW

File: mods/ra/maps/allies-05a/map.yaml [hunk 1]
```diff
@@ -1592,7 +1592,7 @@ Rules:
 		-SpawnMPUnits:
 		-MPStartLocations:
 		LuaScript:
-			Scripts: allies05a.lua, AI.lua
+			Scripts: allies05a.lua, AI05a.lua
```
Comment (pchote) [inline mods/ra/maps/allies-05a/map.yaml]: This becomes ambiguous if/when we have a soviet05a that also has an AI file.  I suggest `allies05a-ai.lua`.

Comment (pchote) [inline mods/ra/maps/allies-05a/map.yaml]: and likewise for all the other files that you refer to only by their mission number.

**Comment by abcdefg30** [comment]
Updated.

**Comment by Mailaender** [comment]
Thanks. :+1:

**Comment by Mailaender** [comment]
Needs a rebase now.

**Comment by abcdefg30** [comment]
Rebased.

**Pull request merged**